cmd/tfimport/README.md

```diff
@@ -0,0 +1,46 @@
+# Terraform Importer
+
+Status: Early Access Program
+
+Terraform Importer is a standalone utility to import existing infrastructure
+into a Terraform state, based on existing Terraform configs. Its target use-case
+is to facilitate migration between other infrastructure-as-code tools (such as
+[Deployment Manager](https://cloud.google.com/deployment-manager)), and configs
+generated by the Terrarom Engine.
+
+## Requirements
+
+- [Terraform 0.12+](https://www.terraform.io/downloads.html)
+- [Go 1.14+](https://golang.org/dl/)
+
+## Usage
+
+```bash
```

Comment:
I used shell in the engine readme, i think for consistency let's stick to one.

Reply:
Done.